app/grandchallenge/core/models.py

```diff
@@ -72,6 +73,14 @@ def status_to_string(self):
             status += ", was accepted at " + self.format_date(self.changed)
         elif self.status == self.REJECTED:
             status += ", was rejected at " + self.format_date(self.changed)
+
+            try:
+                user_is_verified = self.user.verification.is_verified
+            except ObjectDoesNotExist:
+                user_is_verified = False
+
+            if not user_is_verified:
+                status += ". You need to verify your account in order to request access permissions"
```

Comment:
They end up permanently rejected. They should be left in a pending state. Then, if verified accounts are auto accepted for this thing they should be informed to verify their account.

Reply:
Done. Now the message is updated for the requests with pending status only.